iocore/net/QUICNetVConnection.cc

```diff
@@ -1052,8 +1050,8 @@ QUICNetVConnection::_state_handshake_process_handshake_packet(QUICPacketUPtr pac
 {
   // Source address is verified by receiving any message from the client encrypted using the
```

Comment:
Receiving a message encrypted using Handshake key is not the only way. Address validation can be done with address validation tokens. I'm not going to stop landing this, but it should be fixed in the future.

I haven't thought this deeply, but I would make something like these:
- QUICAddressValidator
- QUICPacketSizeLimiter interface
- QUICCongestionController extends QUICPacketSizeLimiter
- QUICAddressValidator extends QUICPacketSizeLimiter
- QUICPathValidator extends QUICPacketSizeLimiter
- QUICPMTUDiscoverer extends QUICPacketSizeLimiter

Reply:
Right. Please file an issue.

Reply:
Filed one for address validation with tokens. #4930